infrastructure/ephemeral-resources/argocd-apps/values.yaml

```
@@ -0,0 +1,28 @@
+# -- Deploy Argo CD Applications within this helm release
+# @default -- `[]` (See [values.yaml])
+## Ref: https://github.com/argoproj/argo-cd/blob/master/docs/operator-manual/
+
+applications:
+- name: nginx-controller
+  finalizers:
+  - resources-finalizer.argocd.argoproj.io
+  project: default
+  sources:
+    - repoURL: https://kubernetes.github.io/ingress-nginx
+      targetRevision: 4.8.3
+      chart: ingress-nginx
+      helm:
+        valueFiles:
+        - $conquer/charts-overrides/nginx-controller/values.yaml
+    - repoURL: https://github.com/conquer-project/conquer_platform.git
+      targetRevision: feat/nginx-ingress-argocd
```

- [ ] Change after approved

```suggestion
      targetRevision: main
```